config/routes.rb

```diff
@@ -21,11 +21,30 @@
 
   get "refresh-session", to: "sessions#refresh", as: :refresh_session
 
-  scope path: ":policy", defaults: {policy: "student-loans"}, constraints: {policy: %r{student-loans}} do
-    constraints slug: %r{#{StudentLoans::SlugSequence::SLUGS.join("|")}} do
-      resources :claims, only: [:show, :update], param: :slug, path: "/"
+  # Used to constrain claim journey routing so only slugs
+  # that are part of a policy’s slug sequence are routed.
+  class RestrictToSequenceSlugs
```

Comment:
I'm a bit iffy about having the class inline in the routing block. Would it be OK to have it up top, so it makes the routes easier to read?

Reply:
Personally I like having the class near to where it is being used as it places it closer to the context in which it needs to be understood.

Reply:
No drama. Happy for it to stay where it is in that case 👍